run.py

```diff
@@ -27,6 +32,16 @@
 ==============================================
 '''
 
+FILE_NOT_FOUND_ERROR = '''
+==============================================
+The issues.db file seems to be missing.
+
+Please create a issues.db file by running:
+
+python run.py
+==============================================
```

Comment:
How does it work? 
To access this message we need to run `python run.py` and the message is telling us the same thing.